modules/history/src/views/full/index.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import style from './style.scss'
+import moment from 'moment'
+
+import { MessagesViewer } from './message'
+import { ConversationPicker } from './conversation'
+
+export default class FullView extends React.Component {
+  constructor(props) {
+    super(props)
+
+    const blob = new Blob([''], { type: 'application/json' })
+    const url = window.URL.createObjectURL(blob)
+
+    const defaultToDate = moment().startOf('day')
+
+    const defaultFromDate = moment()
+      .startOf('day')
+      .subtract(30, 'days')
+
+    this.state = {
+      conversationsInfo: [],
+      messageGroups: [],
+      to: defaultToDate,
+      from: defaultFromDate,
+      currentConvId: null,
+      fileBlob: blob,
+      fileURL: url
+    }
+  }
+
+  threadIdParamName = 'threadId'
+
+  componentDidMount() {
+    this.getConversations(this.state.from, this.state.to)
+    const url = new URL(window.location.href)
+    const threadId = url.searchParams.get(this.threadIdParamName)
+    if (threadId) {
```

Comment:
I suggest using only ThreadId or ConvoId instead of both interchangebly (I know it's already that way in the event, but iy would make it easier to follow in the module)